incydr/cli/cmds/cases.py

```diff
@@ -263,16 +263,29 @@ def bulk_update(ctx: Context, csv: Path):
 
     """
     client = ctx.obj()
+    username_cache = {}
```

Comment:
I wasn't sure where the best place to cache this data would be.  Obviously right now I've cached it within the scope of this specific command.